packages/blade/src/components/Dropdown/DropdownOverlay.web.tsx

```diff
@@ -73,8 +73,10 @@ const DropdownOverlay: WithComponentId<DropdownOverlayProps> = ({ children }): J
 
   React.useEffect(() => {
     if (isOpen) {
+      triggererRef.current?.focus();
```

Comment:
how was the focus working on chrome family browsers then?

Reply:
It's a bug specifically on safari browsers https://razorpay.slack.com/archives/G01B3LQ9H0W/p1676028810932499?thread_ts=1676005222.027229&cid=G01B3LQ9H0W

on chrome, the focus works as expected by default. This on chrome doesn't do anything now because it will already have focus.